app/templates/components/forms/admin/settings/system/mail-settings.hbs

```diff
@@ -0,0 +1,89 @@
+<h3 class="ui header">
+  {{t 'Mail Settings'}}
+</h3>
+<div class="field">
+  <label>
+    {{t 'From Email'}}
+  </label>
+  {{input type='text' name='email_from'}}
+</div>
+<div class="field">
+  <label>
+    {{t 'From Name'}}
+  </label>
+  {{input type='text' name='email_from_name'}}
+</div>
+<div class="ui hidden divider"></div>
+<div class="field">
+  {{ui-radio label=(t 'Disable email') name='mail_setting' value='disable' current='disable' onChange=(action (mut selectedSetting))}}
+</div>
+<div class="field">
+  {{ui-radio label=(t 'SMTP') name='mail_setting' value='smtp' onChange=(action (mut selectedSetting))}}
+</div>
+<div class="{{if (not-eq selectedSetting 'smtp') 'system hidden'}}">
```

Comment:
Don't use a hidden class. Use a full if statement and don't render the things not required.